.github/workflows/test-and-lint.yml

```
@@ -0,0 +1,74 @@
+name: Test and Lint
+
+on:
+  - push
+  - pull_request
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repository code
+        uses: actions/checkout@v2
+
+      - name: Set up node version
+        uses: actions/setup-node@v2
+        with:
+          node-version: '10'
+          cache: 'yarn'
+
+      - name: Install Dependencies
+        run: yarn install --immutable --immutable-cache
+
+      - name: Run lint
+        run: ./node_modules/.bin/standard
```

Even though this isn't an npm script (although maybe it should be), you should be able to do 
```suggestion
        run: yarn standard
```
to invoke a script in `node_modules/.bin`